### PR TITLE
Room add

### PR DESCRIPTION
**Pull request by su-chang**

### CODE REVIEW

**Comment by su-chang** [comment]
Related issue: https://github.com/wechaty/wechaty/issues/2201

File: src/puppet.ts [hunk 1]
```diff
@@ -1005,15 +1005,15 @@ export abstract class Puppet extends PuppetEventEmitter {
...
-  public abstract roomAdd (roomId: string, contactId: string)          : Promise<void>
-  public abstract roomAvatar (roomId: string)                          : Promise<FileBox>
-  public abstract roomCreate (contactIdList: string[], topic?: string) : Promise<string>
-  public abstract roomDel (roomId: string, contactId: string)          : Promise<void>
-  public abstract roomList ()                                          : Promise<string[]>
-  public abstract roomQRCode (roomId: string)                          : Promise<string>
-  public abstract roomQuit (roomId: string)                            : Promise<void>
-  public abstract roomTopic (roomId: string)                           : Promise<string>
-  public abstract roomTopic (roomId: string, topic: string)            : Promise<void>
+  public abstract roomAdd (roomId: string, contactId: string, invite?: boolean) : Promise<void>
```
Comment (windmemory) [inline src/puppet.ts:1008]: Maybe `forceInvite` is better? What do you think @huan

Comment (huan) [reply src/puppet.ts:1008]: This is a super hard question!

Comment (windmemory) [reply src/puppet.ts:1008]: Any idea of this question? @huan

**Review by windmemory — APPROVED** [review]

**Review by huan — DISMISSED** [review]
LGTM

Thank you very much for the improvement!

**Pull request merged**